org.adempiere.base/src/org/compiere/util/Env.java

```diff
@@ -603,6 +605,11 @@ public static String getContext (Properties ctx, String context)
 			if (retValue == null)
 				retValue = System.getProperty(context.substring(PREFIX_SYSTEM_VARIABLE.length()), "");
 			return retValue;
+		} else if (context.startsWith(PREFIX_SYSCONFIG_VARIABLE)) {
+			String retValue = MSysConfig.getValue(context.substring(PREFIX_SYSCONFIG_VARIABLE.length()), Env.getAD_Client_ID(Env.getCtx()), Env.getAD_Org_ID(Env.getCtx()));
```

Comment:
is here good to use org from ctx? Org in context are normally used as default org. 
If organization level system configuration needs to support then organization from window context should use.

Reply:
At this stage, the windowNo variable is not available to access window context. So your suggestion is to remove the AD_Org_ID argument ?

Reply:
Yes, Just remove AD_Org_ID and support only client level or system level configuration.